Assignments/assignment02_lottery/lottery.py

```diff
@@ -38,33 +41,37 @@ def input_lotteryNum():
                     print(MSG_RANGE_ERROR)
                     break
                 elif int(temp[i]) in lotteryNum:
-                    print(MSG_INPUT_OVERLAPPED) 
+                    print(MSG_INPUT_OVERLAPPED)
                     break
-                else: 
+                else:
                     lotteryNum.append(int(temp[i]))
                     count += 1
             if count == 6:
                 break
         except ValueError:
             print(MSG_VALUE_ERROR)
             continue
-    lotteryNum.sort()  
-    return(lotteryNum)
-    
+    lotteryNum.sort()
+    return (lotteryNum)
+
+
 # 몇 장 구입했는지 계산
 def compute_purchase_amount(purchase_price):
     purchase_amount = int(purchase_price) / 1000
     return int(purchase_amount)
 
+
 # 몇 장 구매했는지 출력
 def print_purchase_amount(purchase_amount):
     print("{}장의 로또를 구입하셨습니다.".format(purchase_amount))
 
+
 # 구매한 로또 번호 출력
 def print_lottery(lotteryArr):
     for i in range(len(lotteryArr)):
```

Comment:
위의 방법도 가능하지만 아래 처럼 사용할 수 있을 것 같아요!! list자체를 for문으로 순회하면서 요소들을 출력하면 인덱스로 값들을 가져오지 않아도 될 것 같아요!!!

추가적으로 조금 더 말씀드리자면 변수명이나 method명에 자료형을 넣는 것은 좋은 naming 규칙은 아니에요!!!

```suggestion
    for lottery in lottery_dummy:
        print(lottery)
```

Reply:
앞으로 작명 시 좀 더 주의해야겠습니다.
감사합니다.